docker-compose.yml

```diff
@@ -1,7 +1,27 @@
+version: '3.7'
 services: 
   central-service:
     build: ./central-service
+    depends_on:
+      - db
+    ports:
+      - 5000:5000
+    environment:
+      FIREBASE_CREDENTIALS_FILEPATH: '/app/serviceKey.json'
   rtc-service:
     build: ./rtc-service
   worker-service:
-    build: ./worker-service
\ No newline at end of file
+    build: ./worker-service
+  db: 
+    image: postgres
+    restart: always
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_USER: bsgdev
+      POSTGRES_PASSWORD: bsgdev
```

Comment:
Will need to find a way to access the password securely.

One idea is to store the files in a Google Secrets Manager and have a script pull the secret values from the manager into a .env file. The docker will then read the secret values from the .env file.